apps/src/applab/designElements/library.js

```diff
@@ -149,6 +149,8 @@ export default {
         return ElementType.IMAGE;
       case 'canvas':
         return ElementType.CANVAS;
+      case 'textarea':
```

Comment:
I think that "text areas" generated via the design tab are actually divs -- it looks like the ones generated by this change are real html `textarea` elements. Is that intentional?

Reply:
Thanks for catching this! I was basing the implementation of `textArea` function on `textLabel`. But after you pointed this out, realized that it should be implemented to how [`create`](https://github.com/code-dot-org/code-dot-org/blob/b7e03ff4a27d8bfcb696dce760bf3bb4f7e783d7/apps/src/applab/designElements/textarea.jsx#L186) is defined in `textarea.jsx`.  Updated accordingly - thank you!